### ISSUE TITLE
Score-P C99 compiler detection broken with latest master

### ISSUE THREAD

**Issue by rtschueter** [score-p/scorep_binding_python#150]
Output of Score-P configuration summary slightly changed, therfore compiler detection is currently broken with latest Score-P master, see:
https://github.com/score-p/scorep_binding_python/blob/87f14f86d89592fb9f3f652a026744908c06ba5e/setup.py#L16

**Comment by rtschueter** [comment]
Proposal:
```
diff --git a/setup.py b/setup.py
index 58f216d..7fa1011 100644
--- a/setup.py
+++ b/setup.py
@@ -14,6 +14,11 @@ if not ("shared=yes" in link_mode):
     )
 
 check_compiler = scorep.helper.get_scorep_config("C99 compiler used:")
+if check_compiler is None:
+    check_compiler = scorep.helper.get_scorep_config("C99 compiler:")
+if check_compiler is None:
+    raise RuntimeError("Can not parse the C99 compiler, aborting!")
+
 if "gcc" in check_compiler:
     gcc_plugin = scorep.helper.get_scorep_config("GCC plug-in support:")
     if not ("yes" in gcc_plugin):

```

**Comment by AndreasGocht** [comment]
Please feel free to do a PR.

Best,

Andreas

**Comment by rtschueter** [comment]
Sure, if you would manage appropriate permissions. Thanks.

Am 22.07.2022 um 15:29 schrieb Andreas Gocht ***@***.***>:

﻿

Please feel free to do a PR.

Best,

Andreas

**Comment by AndreasGocht** [comment]
ähhmmm 😅 did I miss some changes in GitHubs rights management? Usually, you simply fork the repo, add your changes, and then start a PR from your fork?

Best,

Andreas

**Comment by maximilian-tech** [comment]
I created a PR.
This fix is needed for Score-P 8.
Thanks @rtschueter for providing a solution!

**Comment by AndreasGocht** [comment]
Closed with #151.

Best

Andreas
